src/stories/Library/search-result-item/SearchResultItem.stories.tsx

```diff
@@ -44,6 +43,10 @@ export default {
         ],
       },
     },
+    availabilityLabels: {
+      control: { type: "null" },
```

Comment:
I wonder if there is a reason the number of availability labels does not use a standard control 🤔 .